decimal64_test.go

```diff
@@ -1,10 +1,10 @@
 package decimal
 
 import (
+	//"fmt"
```

Comment:
Remove the comment. If you're keeping it around for convenience, a properly setup editor will add these automatically so you don't need to.